src/diagnostics/Diagnostic.cpp

```diff
@@ -132,13 +132,15 @@ void
 Diagnostic::TrimIOBox (amrex::Box& box_3d, amrex::Box& domain_3d, amrex::RealBox& rbox_3d)
 {
     if (m_slice_dir >= 0){
+        const amrex::Real half_cell_size = rbox_3d.length(m_slice_dir) /
+                                           ( 2 * domain_3d.length(m_slice_dir) );
+        const amrex::Real mid = (rbox_3d.lo(m_slice_dir) + rbox_3d.hi(m_slice_dir)) / 2;
         // Flatten the box down to 1 cell in the approprate direction.
```

Comment:
Could we clarify that we mean a real number here?
```suggestion
        const amrex::Real half_cell_size = rbox_3d.length(m_slice_dir) /
                                           ( 2. * domain_3d.length(m_slice_dir) );
        const amrex::Real mid = (rbox_3d.lo(m_slice_dir) + rbox_3d.hi(m_slice_dir)) / 2.;
        // Flatten the box down to 1 cell in the approprate direction.
```